tox.ini

```diff
@@ -1,14 +1,16 @@
 [tox]
-envlist = py{36,37,38}-pyqt5, py{36,37,38}-pyside2, linting
+envlist = py{36,37,38}-pyqt5, py{36,37,38}-pyside2, py{37,38}-pyside6, linting
```

Comment:
Can you please also add test environments for this to CI (`.github/workflows/main.yml`)?

Reply:
Changed in 7dd06d7. 
But it seems that only Ubuntu 20.04 is supported by Qt 6. https://www.qt.io/blog/qt6-development-hosts-and-targets